plugins/pulp_rpm/plugins/db/models.py

```diff
@@ -553,6 +554,17 @@ def download_path(self):
         """
         return self.filename
 
+    def verify_size(self, location):
+        """
+        Verify size of the file at the specified location
+
+        :param location: path to the unit location
+        :type  location: str
+        """
+        if self.size is not None:
+            with open(location) as fp:
+                verification.verify_size(fp, self.size)
```

Comment:
I had a look at this function and I find it very bizarre. Can we not just do `os.stat(location).st_size` here?